libpd_wrapper/z_libpd.c

```diff
@@ -79,7 +79,9 @@ static int s_initialized = 0;
 int libpd_init(void) {
   if (s_initialized) return -1; // only allow init once (for now)
   s_initialized = 1;
+#ifndef __EMSCRIPTEN__
```

Comment:
Unfortunately I got to this too late to include this change into the pure-data repo itself.